mobilecoind/README.md

```diff
@@ -46,8 +46,8 @@ This retrieves a json record of:
 
 ```json
 {
-    "enclave": "pool/<git revision>,
-    "sigstruct": "pool/<git revision>,
+    "enclave": "pool/<git revision>",
```

Comment:
For completeness, this should  be:  `"pool/<git revision>/<signing hash>/<filename>"`